.gitattributes

```diff
@@ -0,0 +1,8 @@
+###############################################################################
+# Set default behavior to automatically normalize line endings.
+###############################################################################
+* text=auto
```

Comment:
I think we want only LF line endings, not auto. (I've never seen a programming-oriented text editor on Windows that didn't work just fine with LF-only files.)